yamllint/cli.py

```diff
@@ -207,6 +209,11 @@ def run(argv=None):
     if conf.locale is not None:
         locale.setlocale(locale.LC_ALL, conf.locale)
 
+    if args.list_files:
+        for file in find_files_recursively(args.files, conf):
+            print(file)
```

Comment:
We could strip `./` as well but I opted for the most boring solution.

Reply:
@adrienverge I confirmed that `--list-files` prints the file names in the same format as `yamllint ...`:

Given `/tmp/foo.yaml` with

```yaml
asdf
```

I tested:

```
$ yamllint /tmp/
/tmp/foo.yaml
  1:1       warning  missing document start "---"  (document-start)

$ cd /
$ yamllint --list-files /tmp/
/tmp/foo.yaml

$ cd /tmp
$ yamllint .
./foo.yaml
  1:1       warning  missing document start "---"  (document-start)

$ yamllint --list-files .
./foo.yaml

$ mkdir bar
$ cd bar
$ pwd
/tmp/bar

$ yamllint  ..
../foo.yaml
  1:1       warning  missing document start "---"  (document-start)

$ yamllint --list-files  ..
../foo.yaml
```

Reply:
Perfect!